.github/workflows/k8s-ci.yml

```diff
@@ -390,8 +390,8 @@ jobs:
           # download dataset
           git clone -b master --single-branch --depth=1 https://github.com/7br/gstest.git ${GS_TEST_DIR}
 
-          minikube start --base-image='registry-vpc.cn-hongkong.aliyuncs.com/graphscope/kicbase:v0.0.30' \
-                         --cpus='12' --memory='32000mb' --disk-size='40000mb' \
```

Comment:
@siyuan0322 Could you please help to check if it is ok to don't use the `graphscope/kicbase:v0.0.30` as the base image? Thanks!

Reply:
I have run it locally, and it can't work. 

```
$ minikube start -p test2 --nodes 2 --cpus='12' --memory='32000mb' --disk-size='40000mb' --mount=true --mount-string="/opt/caoye/graphscope/test/gstest:/testingdata" --base-image='registry-vpc.cn-hongkong.aliyuncs.com/graphscope/kicbase:v0.0.30'
```

FYI @siyuan0322

Reply:
This image is necessary as it could greatly reduce the image pulling duration.

Reply:
`registry-vpc` is not accessible from local. It's only accessible from ECS (where the self-hosted machine located in)